notebooks/title_mentions.py

```diff
@@ -0,0 +1,322 @@
+        one_shot_prompt = f1.read()
+    print(one_shot_prompt)
+    return f1, one_shot_prompt
+
+
+@app.cell
+def _(one_shot_prompt, sample_page, submit_prompt):
+    # get response
+    one_shot_response = submit_prompt(
+        task_prompt=one_shot_prompt, user_prompt=sample_page
+    )
+    return (one_shot_response,)
+
+
+@app.cell
+def _(one_shot_response):
+    print(one_shot_response)
+    print()
+    print(one_shot_response.choices[0].message.content)
+    return
+
+
+@app.cell
+def _(display_bracketed_response, one_shot_response):
+    # Finds the texts of interest but does not include volume-level information.
+    # Misses the additional speech (https://www.marxists.org/deutsch/archiv/marx-engels/1864/10/inaugadr.htm) in the first paragraph
+    display_bracketed_response(one_shot_response)
+    return
+
+
+@app.cell
+def _(mo):
+    mo.md(
+        """
+        ## Switching to smaller text chunks
+        Prompting with a full page appears to take a fairly long time (~30s), so explore response times for smaller chunks.
+
+        **Results: Faster responses for smaller text chunks.**
+        """
+    )
+    return
+
+
+@app.cell
+def _(sample_page):
+    text_chunks = sample_page.split("\n\n")
+    return (text_chunks,)
+
+
+@app.cell
+def _(basic_prompt, display_bracketed_response, submit_prompt, text_chunks):
+    # Skipped speech
+    display_bracketed_response(
+        submit_prompt(task_prompt=basic_prompt, user_prompt=text_chunks[0])
+    )
+    return
+
+
+@app.cell
+def _(basic_prompt, display_bracketed_response, submit_prompt, text_chunks):
+    display_bracketed_response(
+        submit_prompt(task_prompt=basic_prompt, user_prompt=text_chunks[1])
+    )
+    return
+
+
+@app.cell
+def _(basic_prompt, display_bracketed_response, submit_prompt, text_chunks):
+    # Skips volume information
+    display_bracketed_response(
+        submit_prompt(task_prompt=basic_prompt, user_prompt=text_chunks[2])
+    )
+    return
+
+
+@app.cell
+def _(basic_prompt, display_bracketed_response, submit_prompt, text_chunks):
+    # Skips volume information
+    display_bracketed_response(
+        submit_prompt(task_prompt=basic_prompt, user_prompt=text_chunks[3])
+    )
+    return
+
+
+@app.cell
+def _(mo):
+    mo.md(
+        r"""
+        ## Conclusions
+        - Volume-level information is generally ignored.
+        - Prompting to use square brackets for annotations appears to work, will be relatively easy to convert for viewing assuming the underlying text does not include square brackets.
+        - Not clear that adding an example to the prompt helps.
+        - The newer prompt template appears to take a lot more time, but it seems to be relative to the length of the input passage.
+        - Annotations do not appear to be particularly stable between runs.
+            - Spurious annotations may appear in passages without any title references
+            - The speech written by Karl Marx is not necessarily identified
```

Comment:
Did you mean text here?
```suggestion
            - The text written by Karl Marx is not necessarily identified
```

Reply:
It's a speech, an [inaugural address](https://www.marxists.org/deutsch/archiv/marx-engels/1864/10/inaugadr.htm)